src/pages/profiles/Profile.vue

```diff
@@ -183,7 +184,10 @@ export default {
         first: 5,
         offset: 0,
         fetchMore: true
-      }
+      },
+
+      proposals: [],
```

Comment:
Maybe change the variable name to multiSigProposals so we don't get confused with the actual proposals